example/storybook/stories/BrandConfigProvider/BrandConfigProvider.stories.tsx

```diff
@@ -16,7 +16,7 @@ storiesOf('BrandConfigProvider', module)
     const customColors = object('theme.colors', baseTheme.colors);
     const customSpacing = object('theme.spacing', baseTheme.spacing);
```

Comment:
I could be wrong about this but having only spacing defined in our theme feels like it is not granular enough to be useful. For example, if we are using `spacing.medium` globally for things like margin, padding, etc, won't it be total mayhem if that value is tweaked? Perhaps splitting these out by usage would be helpful, i.e., `margin.small` or `padding.medium` to get more predicable results could help? Even still I am having a hard time imaging a scenario where tweaking these global spacing values would have the desired outcome, almost feels like these should just be constants excluded from our theming.

Reply:
Yes, I agree 💯 that it isn't granular enough to be useful.  I would be in favor of removing it.  It isn't a part of MD either, and Paper seems to just have sensible and opinionated defaults in density-independent pixels that work for most apps and allow styling to override it.

This is interesting: https://m3.material.io/foundations/layout/understanding-layout/spacing

But yea, I'm in favor of removing spacing.

Reply:
Great point @bruce-glazier. Probably best to just bake those into the default styles and let a consumer tweak them if they want.